src/app/base/components/node/MachinesHeader/MachinesHeader.tsx

```diff
@@ -7,15 +7,13 @@ import { matchPath, Link } from "react-router-dom-v5-compat";
 
 import type { SectionHeaderProps } from "app/base/components/SectionHeader";
 import SectionHeader from "app/base/components/SectionHeader";
-import machineURLs from "app/machines/urls";
-import poolsURLs from "app/pools/urls";
+import urls from "app/base/urls";
```

Comment:
It seems like you've used a mix of the top level urls and section URLs. If we're going to have a top level urls file do you think that we should use it for everything (that could be a followup if it'd make this PR to large).

Reply:
I did, I didn't want to make this PR to big, will follow up 👍

Reply:
https://github.com/canonical-web-and-design/maas-ui/issues/4165